local-dev/templates/docker-rsync-local-rawls.sh

```diff
@@ -126,9 +126,9 @@ start_server () {
     -e PROXY_URL='http://rawls-sbt:8080/' \
     -e PROXY_URL2='http://rawls-sbt:8080/api' \
     -e PROXY_URL3='http://rawls-sbt:8080/register' \
-    -e CALLBACK_URI='https://local.broadinstitute.org/oauth2callback' \
+    -e CALLBACK_URI='https://local.dsde-dev.broadinstitute.org/oauth2callback' \
```

Comment:
I've replaced `local.broadinstitute.org` with `local.dsde-dev.broadinstitute.org` everywhere in the code - please tell me if any of these need to stay `local.broadinstitute.org`!